not_so_basic_grunt.py

```diff
@@ -0,0 +1,100 @@
+import pygame
+import math
+
+from gameobject import GameObject
+
+
+class NotSoBasicGrunt(GameObject):
+    
+
+    def __init__(self, starting_position, universe):
+        self.position_ = starting_position
+
+        self.ID_ = self.create_ID()
+        self.size_ = 15
+        self.health_ = 30
+        self.universe_ = universe
+        self.listeners_ = []
+        self.enemy_type_ = "NotSoBasicGrunt"
+
+        self.x_velocity = 0
+        self.y_velocity = 0.3 
+        self.x_accel = 0
```

Comment:
Normally I'd say abbreviations are not good in variable names, but I think this one is good. `acceleration` is a long word.